tests/merge_test.py

```diff
@@ -17,30 +17,32 @@ def __tmp_paths(tmp_path):
     return [tmp_path / p for p in ["rpki_final.txt", "irr_final.txt", "out.txt"]]
 
 def __read_test_vectors(filepath):
-    """
+    '''
     Fixtures for IP networks are under tests/data.
-    Read them and return the list of valid networks and the count of individual subnets expected in the result of the merge.
-    """
+    Read them and return the list of valid networks and the list of individual subnets in the file.
+    '''
     networks = []
-    subnet_count = 0
+    subnets = []
     with open(filepath, "r") as f:
         lines = f.readlines()[1:]
         for line in lines:
             network, is_valid, is_subnet = line.split(',')
             if is_valid == "TRUE":
                 networks.append(network)
             if is_subnet.strip() == "TRUE":
-                subnet_count += 1
-    return networks, subnet_count
+                subnets.append(network)
+    return networks, subnets
 
 def test_merge_from_fixtures(tmp_path):
     '''
-    Assert that general_merge merges subnets correctly.
+    Assert that general_merge merges subnets correctly,
+    and validates against expected network sets, i.e. invalid networks are not merged,
+    and subnets are merged into the root network appropriately.
     '''
 
-    base_nets, base_subnet_count = __read_test_vectors("tests/data/base_file.csv")
+    base_nets, base_subnets = __read_test_vectors("tests/data/base_file.csv")
```

Comment:
Hm, should (here and elsewhere in the file) `subnet` be renamed to something more general like `exludeds` or so? Because it also includes invalids that are excluded even if they may not be a subnet. And also when we have duplicates they are also not included, right?

Reply:
:+1: done